run-checks

```diff
@@ -258,14 +264,23 @@ if [ "$UNIT" = 1 ]; then
     fi
 fi
 
-if [ "$SPREAD" = 1 ]; then
+if [ "$SPREAD" -gt 0 ]; then
     TMP_SPREAD="$(mktemp -d)"
     addtrap "rm -rf \"$TMP_SPREAD\""
 
     export PATH=$TMP_SPREAD:$PATH
     ( cd "$TMP_SPREAD" && curl -s -O https://niemeyer.s3.amazonaws.com/spread-amd64.tar.gz && tar xzvf spread-amd64.tar.gz )
 
-    spread google: google-upgrade:tests/upgrade/
+    if [ "$SPREAD" = full ]; then
```

Comment:
Perhaps this should be a switch statement?

Reply:
Yes, thanks for the review!!